pytest.ini

```diff
@@ -1,5 +1,4 @@
-# Setup for pytest
 [pytest]
 markers =
     mat_ops: mark a test as utilizing matrix operations.
-addopts = --durations=10
+addopts = --durations=10 --doctest-modules --doctest-continue-on-failure --showlocals
```

Comment:
Let's leave out continue on failure.

Reply:
What is the story behind that?  

Usually, it's great to see all problems at once.